akka-projection-eventsourced/src/main/scala/akka/projection/eventsourced/javadsl/EventSourcedProvider.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.projection.eventsourced.javadsl
+
+import java.util.Optional
+
+import akka.NotUsed
+import akka.actor.ClassicActorSystemProvider
+import akka.persistence.query.javadsl.EventsByTagQuery
+import akka.persistence.query.NoOffset
+import akka.persistence.query.Offset
+import akka.persistence.query.PersistenceQuery
+import akka.projection.eventsourced.EventEnvelope
+import akka.projection.javadsl.SourceProvider
+import akka.stream.javadsl.Source
+import com.typesafe.config.Config
+import com.typesafe.config.ConfigFactory
+
+object EventSourcedProvider {
+
+  def eventsByTag[Event](
+      systemProvider: ClassicActorSystemProvider,
+      readJournalPluginId: String,
+      tag: String): SourceProvider[Offset, EventEnvelope[Event]] =
+    eventsByTag(systemProvider, readJournalPluginId, ConfigFactory.empty(), tag)
+
+  def eventsByTag[Event](
+      systemProvider: ClassicActorSystemProvider,
+      readJournalPluginId: String,
+      readJournalPluginConfig: Config,
```

Comment:
The config variant is some power user api that I'm not sure about, let's skip that until there is a clear need for it.